mapit_global/management/commands/mapit_global_import.py

```diff
@@ -203,7 +203,7 @@ def verbose(s):
                 if len(layer) != 1:
                     raise Exception("We only expect one feature in each layer")
 
-                feat = layer[1]
+                feat = next(iter(layer))
```

Comment:
A side note here – I get a notification from github for a commit like this whenever it's rebased, edited, changed. So I'd prefer not to @ me in commit messages, please, unless you know never going to be altered ;)